astropy/wcs/utils.py

```diff
@@ -95,10 +96,50 @@ def _wcs_to_celestial_frame_builtin(wcs):
     return frame
 
 
+def _celestial_frame_to_wcs_builtin(frame, projection=None):
```

Comment:
Why not just set the default projection to `TAN` here? (I think one should use `None` only if the value depends on other arguments or state)